assets/config/bootstrap-variables.scss

```diff
@@ -234,7 +234,7 @@ $font-family-base:            $font-family-sans-serif !default;
 
 $font-size-base:              1rem !default; // Assumes the browser default, typically `16px`
 $font-size-lg:                ($font-size-base * 1.25) !default;
-$font-size-sm:                ($font-size-base * .83) !default;
+$font-size-sm:                0.778rem !default;
```

Comment:
`1rem * 0.83 ≠ 0.778rem` 😅  Is it intended to reduce this value ?

Reply:
My bad, good catch!

Reply:
Ah but that was the value used in .text-small, I figured this would be better to be applied to all "sm" texts. Is it an issue in some cases?

Reply:
I think we are using a lot `$font-size-sm` in our styles 🤔 I prefer `.text-small` to be equal `0.83rem`, but we must be sure that all components who use that class will not break.

Reply:
👍